src/components/Books.jsx

```diff
@@ -0,0 +1,21 @@
+import Form from './Form';
+import Button from './shared/Button';
+
+const Books = () => (
+  <div>
+    <h2>List of Books</h2>
+    <ul>
+      <li>
+        Fight Hunger by Nim
+        <Button value="Remove" />
+      </li>
```

Comment:
- [ ] Let's set up this component to render the Book component with the book data passed in as a prop. See example

  ```
  import Book from './Book';
  import Form from './Form';
  import Button from './shared/Button';
  
  const Books = () => (
    <div>
      <h2>List of Books</h2>
      <ul>
        <li>
          <Book title="first book" author="first author" />
        </li>
  .
  .
  .
  ```